cmd/yggd/main.go

```diff
@@ -377,6 +377,9 @@ func main() {
 		env := []string{
 			"YGG_SOCKET_ADDR=unix:" + c.String("socket-addr"),
 			"PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
+			"HTTP_PROXY=" + os.Getenv("HTTP_PROXY"),
```

Comment:
The history of these environment variable names is messy. `HTTP_PROXY` might be supported by some implementations, but the "curl standard" is that `http_proxy` is lower-case, so we should include the lower and upper-case values.

https://everything.curl.dev/usingcurl/proxies/env
```suggestion
			"HTTP_PROXY=" + os.Getenv("HTTP_PROXY"),
			"http_proxy"= + os.Getenv("http_proxy"),
```

Reply:
And we should do the same for `https_proxy` environment variable. And probably also for `no_proxy`  too.

Reply:
The issue is just with HTTP_PROXY that can be used in CGI environments, the article explains why the decision of curl.